src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -91,55 +93,67 @@ export const MarketplaceSelect = () => {
   }))
 
   const [isOpen, setOpen] = useState(!!selectedMarkets.length)
+  const setTraitsOpen = useTraitsOpen((state) => state.setTraitsOpen)
 
   return (
-    <Box
-      as="details"
-      className={clsx(subheadSmall, !isOpen && styles.rowHover, isOpen && styles.detailsOpen)}
-      borderRadius="12"
-      open={isOpen}
-    >
+    <>
+      <Box className={styles.detailsOpen} opacity={isOpen ? '1' : '0'} />
       <Box
-        as="summary"
-        className={clsx(isOpen && styles.summaryOpen, isOpen ? styles.rowHoverOpen : styles.rowHover)}
-        display="flex"
-        justifyContent="space-between"
-        cursor="pointer"
-        alignItems="center"
-        fontSize="14"
-        paddingTop="12"
-        paddingLeft="12"
-        paddingRight="12"
-        paddingBottom={isOpen ? '8' : '12'}
-        onClick={(e) => {
-          e.preventDefault()
-          setOpen(!isOpen)
-        }}
+        as="details"
+        className={clsx(subheadSmall, !isOpen && styles.rowHover)}
+        open={isOpen}
+        borderRadius={isOpen ? '0' : '12'}
       >
-        Marketplaces
         <Box
-          color="textSecondary"
-          transition="250"
-          height="28"
-          width="28"
-          style={{
-            transform: `rotate(${isOpen ? 0 : 180}deg)`,
+          as="summary"
+          className={clsx(isOpen ? styles.rowHoverOpen : styles.rowHover)}
+          display="flex"
+          justifyContent="space-between"
+          cursor="pointer"
+          alignItems="center"
+          fontSize="16"
+          paddingTop="12"
+          paddingLeft="12"
+          paddingRight="16"
+          paddingBottom="12"
+          lineHeight="20"
+          borderRadius="12"
+          maxHeight="48"
+          onClick={(e) => {
+            e.preventDefault()
+            setOpen(!isOpen)
+            setTraitsOpen(-10, !isOpen)
```

Comment:
I don't want to show two lines or lines to double up (making them thicker) when the previous filter is open.  For the price range filter, I need to know when market is "open".  The dynamic filters are set based on their index and they check the previous index to see if it is open.  Therefore, index 0 checks index -1.  I didn't want to have separate states, so that is why this is a rounded -10.